test/python/transpiler/test_optimize_annotated.py

```diff
@@ -101,7 +102,9 @@ def test_optimize_definitions(self):
         qc.h(0)
         qc.append(gate, [0, 1, 3])
 
-        qc_optimized = OptimizeAnnotated(basis_gates=["cx", "u"])(qc)
+        # Add "swap" to the basis gates to prevent conjugate reduction from replacing
+        # control-[SWAP] by CX(0,1) -- CCX(1, 0) -- CX(0, 1)
```

Comment:
That's fantastic emergent behaviour! I never would have thought of that, but it's totally obvious as soon as you see the computer do it.

Reply:
I have also added a more explicit test for this.